partcad/src/partcad/project_factory_git.py

```
@@ -86,80 +137,114 @@
+                                origin = repo.remote("origin")
+                                # Need to check for updates
+                                origin.fetch()
+                                repo.git.checkout(self.import_revision, force=True)
+                                origin.pull(force=True, rebase=True)
+                                self.ctx.stats_git_ops += 1
+                                os.utime(guard_path, (now, now))
+                            else:
+                                # No update was performed
+                                before = None
+
+                        if not before is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)